core/vtk/ttkScalarFieldSmoother/ttkScalarFieldSmoother.cpp

```diff
@@ -53,6 +53,14 @@ int ttkScalarFieldSmoother::RequestData(vtkInformation *ttkNotUsed(request),
 
   this->preconditionTriangulation(triangulation);
 
+#if TTK_ENABLE_MPI
+
+  this->MPIPreconditioning(input);
```

Comment:
filters should not call this function.
this should be called in ttkAlgorithm already (are you sure you've updated this branch?)

Reply:
ah yes, I wasn't sure if it was necessary to call it myself, but it's called in ttkAlgorithm so it works without